### PR TITLE
add the domain subtasks: create and delete

### PR DESCRIPTION
**Pull request by sidler-mozilla**

### CODE REVIEW

File: dodo.py [hunk 1]
```diff
@@ -558,6 +558,48 @@ def task_pytest():
...
+        'actions': [
+            'echo "missing valid AWS credentials"',
+            'false',
+        ],
+        'uptodate': [
+            uptodate
+        ],
+    }
+
+def task_domain():
```
Comment (marty331) [inline dodo.py]: task_domain should not delete domain unless specified.

Comment (sidler-mozilla) [reply dodo.py]: impossible to achieve as it is currently configured

**Comment by bowlofstew** [comment]
Do we need to add a task around the existing serverless task?  It seems heavyweight.

**Comment by sidler-mozilla** [comment]
will rewrite another way

**Pull request closed without merging**